src/PrestaShopBundle/Form/Admin/Sell/Product/Image/ImageDropzoneType.php

```diff
@@ -58,7 +58,7 @@ public function configureOptions(OptionsResolver $resolver)
                 'window.replaceSelection' => $this->trans('Replace selection', 'Admin.Actions'),
                 'window.cantDisableCover' => $this->trans('Using another image as cover will automatically uncheck this box.', 'Admin.Catalog.Help'),
                 'window.selectedFiles' => $this->trans(
-                    '[1]%filesNb%[/1] selected file(s)',
```

Comment:
https://vue-i18n.intlify.dev/guide/migration/breaking.html#apis

Custom formatter is not supported anymore until 10.x version of vue-i18n, it would add too much complexity to fix it for now, we can revert when 10.x is ready probably

Reply:
OK no other choice then